.github/workflows/release-baseline-images.yml

```diff
@@ -0,0 +1,40 @@
+# Upload the ZIP archive of baseline images as a release asset
+
+name: Upload baseline images
+
+# Only run for releases
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  upload-baseline:
+    name: Upload baseline images
+    runs-on: ubuntu-latest
+    if: github.repository == 'GenericMappingTools/gmt'
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3.0.2
+
+    - name: Setup data version control (DVC)
+      uses: iterative/setup-dvc@v1.0.3
+
+    - name: Pull baseline image data from dvc remote
+      run: |
+        dvc pull
+        ls -lhR test/baseline/
```

Comment:
I think we also need the images in `doc/examples` and `doc/scripts`

Reply:
These are included in the release tarball after https://github.com/GenericMappingTools/gmt/pull/6533, but I added them to the baseline-images asset as well in https://github.com/GenericMappingTools/gmt/pull/6782/commits/aa93c0693d8decf6ae0b41fd3a63663e7ec12d69. This change increases the size from ~40 MB to ~70 MB.